webapp/src/pages/applicationForm/components/ApplicationForm.js

```diff
@@ -1146,13 +1149,19 @@ class ApplicationForm extends Component {
     if (isError) {
       return <div className={"alert alert-danger alert-container"}>{errorMessage}</div>;
     }
-
-    if (formSpec.nominations && responses.length > 0 && !responseSelected) {
+    
+    if (this.props.event.event_type === 'JOURNAL' && journalSubmissionFlag) {
+      return <ApplicationFormInstance
+      formSpec={formSpec}
+      response={selectedResponse}
```

Comment:
Probably better to set response to `null` here to make it more obvious that you're doing a new response and not editing an existing one.

Reply:
Done!